src-ghc/Pact/Interpreter.hs

```diff
@@ -78,12 +78,13 @@ data MsgData = MsgData {
   mdData :: !Value,
   mdStep :: !(Maybe PactStep),
   mdHash :: !Hash,
-  mdSigners :: [Signer]
+  mdSigners :: [Signer],
+  mdSessionSigner :: Maybe Signer
```

Comment:
@larskuhtz, @emilypi  , I've added a field to `MsgData`, and this impacts the hashes of Continuations. (I had to update the golden tests for these, as you can see elsewhere in the PR, e.g. https://github.com/kadena-io/pact/pull/1171/files#diff-970731ecec6c7d8081d89680725cb29612cac4c0eb07557398a40a0f92b99949R4).

Does this mean that my change would be forking for chainweb? Having some trouble tracking down what data ends up in the hashes.

Reply:
Thanks for the head's up. This will fork the interpeter if i'm not mistaken - @jmcardon can you confirm?

Reply:
You can also answer this question by running a replay on your own machine. Since that's a great habit to get into, may I recommend to you the steps in this document, which would also be relevant here: https://www.notion.so/kadenateam/HOWTO-Debug-Gas-Discrepancies-6c3a8c94b56746c7907c582aed5b9ad3?pvs=4